src/Sri.php

```diff
@@ -32,6 +32,10 @@ public function html(string $path, bool $useCredentials = false): string
 
     public function hash(string $path): string
     {
+        if (! $this->isInEnabledEnv()) {
+            return '';
```

Comment:
Is an empty string a valid integrity hash? Will it be skipped?

I'm worried that it could potentially do more harm as it could still try to match an `''` hash against the file.

Reply:
Based from what I've done as a work around in #48, it works on Chrome.